facturacion_mexico/one_offs/backfill_folio_fiscal.py

```
@@ -0,0 +1,103 @@
+		return row.fm_uuid.strip()
+	return ""
+
+
+def run(apply=0):
+	"""Backfill de fm_folio_fiscal. apply=0 (default) → dry-run; apply=1 → escribe."""
+	apply = int(apply)
+	stats = {"revisadas": 0, "actualizadas": 0, "limpiadas": 0, "sin_cambio": 0, "errores": 0}
+	ejemplos = {"actualizadas": [], "limpiadas": [], "sin_cambio": []}
+	MAX_EJEMPLOS = 3
+
+	def _muestra(categoria, name, current, desired):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_muestra`

Add return type annotation: `None`

(ANN202)